src/pages/Examples.tsx

```diff
@@ -1,14 +1,65 @@
+import Block from '@dojo/framework/widget-core/meta/Block';
 import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
 import { tsx } from '@dojo/framework/widget-core/tsx';
 
+import FontAwesomeIcon from '../widgets/icon/FontAwesomeIcon';
+import Grid from '../widgets/grid/Grid';
+import Landing from '../widgets/Landing';
+import LinkedCard from '../widgets/card/LinkedCard';
+import Subsection from '../widgets/section/Subsection';
+import getExamples, { ExampleMeta } from '../scripts/examples.block';
+
 import * as css from './Examples.m.css';
 
 export default class Examples extends WidgetBase {
+	private renderCardHeader = ({ exampleName }: ExampleMeta) => (
+		<div
+			classes={css.header}
+			styles={{
+				backgroundImage: `url(https://github.com/dojo/examples/raw/master/${exampleName}/preview.png)`
+			}}
+		/>
+	);
+
+	private renderCardFooter = ({ exampleName, sandbox }: ExampleMeta) => (
+		<div classes={css.footer}>
+			{sandbox && (
+				<a
+					href={`https://codesandbox.io/s/github/dojo/examples/tree/master/${exampleName}`}
+					classes={css.linkBtn}
+				>
+					<FontAwesomeIcon icon="laptop-code" />
+				</a>
+			)}
+			<a href={`https://github.com/dojo/examples/tree/master/${exampleName}`} classes={css.linkBtn}>
+				<FontAwesomeIcon icon="code-branch" />
+			</a>
+		</div>
+	);
+
 	protected render() {
+		const examples = (this.meta(Block).run(getExamples)() as any) as ExampleMeta[];
```

Comment:
I think this can be `null` too as meta's will return nothing until the block as resolved with the value.

Reply:
I'm surprised this doesn't break though, if it does return null with the `.map` below.

Reply:
That probably explains why I thought the dev server was broken sometimes.